web/frontend/src/app/component/concept-relationship/concept-relationship.component.html

```diff
@@ -359,7 +368,7 @@
       None
     </ng-template>
   </p-panel>
-  <p-panel [toggleable]="true">
+  <p-panel [toggleable]="true" *ngIf="terminology=='ncit'">
```

Comment:
Peter, for these things, what we really want to say is "terminology!='ncim'" because "ncit" is actually the _normal_ case and any other new terminologies will follow that pattern.  Do me a favor and look for all instances of "ncit" in the code and if it's something like this that's about ncit vs ncim functionality, change the predicate to be _not equals_ "ncim" rather than _equals_ "ncit";

Reply:
Ok I modified these instances to check for ncim instead. A couple of them though deal with text that makes explicit reference to NCIt, so I modified those to check each terminology.